ConsoleUI/KSPAddScreen.cs

```diff
@@ -9,7 +13,14 @@ public class KSPAddScreen : KSPScreen {
         /// Initialize the Screen
         /// </summary>
         /// <param name="mgr">KSP manager containing the instances</param>
-        public KSPAddScreen(KSPManager mgr) : base(mgr) { }
+        public KSPAddScreen(KSPManager mgr) : base(mgr)
+        {
+            AddObject(new ConsoleLabel(
+                labelWidth, pathRow + 1, -1,
+                () => $"Example: {Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), "Kerbal Space Program")}",
```

Comment:
Apparently `Environment.SpecialFolder.ProgramFiles` is an empty string on Linux:
```
Example: Kerbal Space Program
```
We can probably expect Linux users to know where their stuff is installed (or at least know where to find it), but it looks kinda weird like this. Maybe we can fall back to the standard steam path or so?

Reply:
Yeah, it doesn't look like Mono was particularly careful in its implementation of these special folders. Some of them make sense and others seem like whatever random choice they felt like making at the time.

https://github.com/mono/mono/blob/09307e6411ace79740781df7a34f8fe09ad98f9f/mcs/class/corlib/System/Environment.cs#L621-L747

Now if only the `??` operator could handle empty strings...

Reply:
Alright, now if Program Files isn't defined, it should fall back to `~/.local/share`.